packages/fscomponents/src/components/EmailForm.tsx

```diff
@@ -6,31 +6,33 @@ import * as t from 'tcomb-form-native';
 import { SingleLineForm } from './SingleLineForm';
 import { FormLabelPosition } from './Form';
 import FSI18n, { translationKeys } from '@brandingbrand/fsi18n';
+import {Dictionary} from '@brandingbrand/fsfoundation';
+import {StyleProp, ViewStyle} from 'react-native';
 const componentTranslationKeys = translationKeys.flagship.emailForm;
 
 export interface EmailFormValue {
   email: string;
 }
 
 export interface EmailFormProps {
-  fieldsStyleConfig?: any;
+  fieldsStyleConfig?: Dictionary;
   labelPosition?: FormLabelPosition;
   onSubmit?: (value: EmailFormValue) => void;
-  submitButtonStyle?: any;
-  submitTextStyle?: any;
-  submitText?: any;
+  submitButtonStyle?: StyleProp<ViewStyle>;
+  submitTextStyle?: StyleProp<ViewStyle>;
+  submitText?: StyleProp<ViewStyle>;
```

Comment:
`submitTextStyle` should be `StyleProp<TextStyle>` and `submitText` should be `Text`. These types come from here: https://github.com/brandingbrand/flagship/blob/develop/packages/fscomponents/src/components/SingleLineForm.tsx ; ideally we should clean up the types on that component and then reference them here.